table/partitioned_filter_block_test.cc

```diff
@@ -66,12 +66,13 @@ class PartitionedFilterBlockTest : public testing::Test {
   }
 
   uint64_t MaxFilterSize() {
+    uint32_t dont_care1, dont_care2;
```

Comment:
I got that you don't need the values returned. What I don't understand is, why moving the declaration of the two values? Is it related to valgrind and asan failure?

In the case that we are sure the type, I suggest we use static_cast<>. I have a long term hope that we get ride of all the dynamic cast in the code base, so that we can disable RTTI if users want to safe some memory: https://google.github.io/styleguide/cppguide.html#Run-Time_Type_Information__RTTI_